packages/block-editor/src/store/reducer.js

```diff
@@ -656,6 +679,54 @@ export const blocks = flow(
 
 		return state;
 	},
+
+	// This is the opposite of the order property.
```

Comment:
I feel we could be more specific with this comment, particularly around what "performance reasons" there are.

```suggestion
	// While technically redundant data as the inverse of `order`, it serves as
	// an optimization for the selectors which derive the ancestry of a block.
```